mandatory/3-function-output.js

```diff
@@ -1,16 +1,32 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
+  /* step 1
+  Math.random() = gives you random number between 0 to 1.
+
+  step 2
+ result of step one multiply by 10 
+  
+  */
+
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
+  /*
+this is similar to word1 + word2;
+
+variable1.concat(varialbe1) returns the comination of the two varialbles
+while the first first varialble will be the first in the sentence.
+*/
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  return firstWord.concat(" ", secondWord, " ", thirdWord);
+  // return concatnatedTrimedwords.trim();
 }
 
 /* 
```

Comment:
Hi khadar good job but in mandatoy folder should be four exercises but in your pull request there are three exercises and you need to add mandatory 4-tax.js